cirq/google/calibration/phased_fsim.py

```diff
@@ -0,0 +1,118 @@
+from typing import Dict, Optional, TYPE_CHECKING
+
+import dataclasses
+
+if TYPE_CHECKING:
+    # Workaround for mypy custom dataclasses
```

Comment:
Can you just add the mypy tracking issue (https://github.com/python/mypy/issues/5406) to this comment?

Reply:
Done.